packages/next-css/index.js

```diff
@@ -12,7 +12,7 @@ module.exports = (nextConfig = {}) => {
       }
 
       const { dev, isServer } = options
-      const { cssModules } = nextConfig
+      const { cssModules, cssLoaderOptions } = nextConfig
```

Comment:
I guess we can just update next-css/next-sass/next-less to pass the `nextConfig` to `cssLoaderConfig` instead of manually passing it in all modules.